code/modules/mob/living/simple_animal/corpse.dm

```diff
@@ -272,58 +250,7 @@
 	suit = /obj/item/clothing/suit/jacket/straight_jacket
 	head = /obj/item/reverse_bear_trap
 
-/obj/effect/mob_spawn/human/corpse/heavy
-	name = "Heavy gunner"
-	brute_damage = 300
-	hair_style = "Bald"
-	facial_hair_style = "Shaved"
-	mob_gender = MALE
-	skin_tone = "caucasian1"
-	outfit = /datum/outfit/minigunheavy
-
-/datum/outfit/minigunheavy
-	name = "Heavy gunner"
-	uniform = /obj/item/clothing/under/rank/security/head_of_security/alt
-	shoes = /obj/item/clothing/shoes/combat/swat
-	gloves = /obj/item/clothing/gloves/combat
-	mask = /obj/item/clothing/mask/gas/sechailer/swat/emagged
-	suit = /obj/item/clothing/suit/armor/heavy
-	back = /obj/item/minigunpack
```

Comment:
And this is the minigun backpack the heavy gunner should not spawn with, and should still be removed. 

It's broken shitcode I fixed up once, but it really doesn't need to exist for players to use.